diff -up dnf-3.0.2/dnf/base.py.omv~ dnf-3.0.2/dnf/base.py
--- dnf-3.0.2/dnf/base.py.omv~	2018-07-03 13:15:24.749391648 +0200
+++ dnf-3.0.2/dnf/base.py	2018-07-03 13:15:35.552390864 +0200
@@ -37,6 +37,7 @@ from dnf.db.history import SwdbInterface
 from dnf.yum import misc
 from functools import reduce
 import collections
+import collections.abc
 import datetime
 import dnf.callback
 import dnf.comps
@@ -835,7 +836,7 @@ class Base(object):
 
     def do_transaction(self, display=()):
         # :api
-        if not isinstance(display, collections.Sequence):
+        if not isinstance(display, collections.abc.Sequence):
             display = [display]
         display = \
             [dnf.yum.rpmtrans.LoggingTransactionDisplay()] + list(display)
diff -up dnf-3.0.2/dnf/cli/cli.py.omv~ dnf-3.0.2/dnf/cli/cli.py
--- dnf-3.0.2/dnf/cli/cli.py.omv~	2018-07-03 13:14:00.491397765 +0200
+++ dnf-3.0.2/dnf/cli/cli.py	2018-07-03 13:14:53.216393937 +0200
@@ -29,6 +29,7 @@ from dnf.cli import CliError
 from dnf.i18n import ucd, _
 
 import collections
+import collections.abc
 import dnf
 import dnf.cli.commands
 import dnf.cli.commands.autoremove
@@ -228,7 +229,7 @@ class BaseCli(dnf.Base):
         if self.conf.downloadonly:
             return
 
-        if not isinstance(display, collections.Sequence):
+        if not isinstance(display, collections.abc.Sequence):
             display = [display]
         display = [output.CliTransactionDisplay()] + list(display)
         super(BaseCli, self).do_transaction(display)
